Github CI is failing.

Tests appear to be failing for recent PRS, npm test abort with following message

```
 >mocha test/*

sh: 1: mocha: not found
npm ERR! Test failed.  See above for more details.
##[error]Process completed with exit code 1.
```

--- a/.github/workflows/nodejs.yml
+++ b/.github/workflows/nodejs.yml
@@ -24,6 +24,7 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
+    - run: npm install mocha
     - run: npm test
       env:
         CI: true

